courses/models_test.py

```diff
@@ -841,3 +842,32 @@ def test_course_language_prevent_delete():
     course_page = CoursePageFactory.create()
     with pytest.raises(ProtectedError):
         course_page.language.delete()
+
+
+@pytest.mark.parametrize(
+    ("is_external", "page_factory"),
+    [(True, ExternalCoursePageFactory), (False, CoursePageFactory)],
+)
+def test_is_external_field_update_after_course_is_attached_with_course_page(
```

Comment:
nit: A better test name could be
```suggestion
def test_prevent_is_external_update_after_course_page_attachment(
```